models/signature.js

```diff
@@ -78,6 +61,75 @@ function parseSignatures(data) {
    return signatures;
 }
 
+/**
+ * Parses a GitHub review and returns an array of Signature objects.
+ * Returns an empty array if the comment did not contain any of our tags.
+ */
+Signature.parseReview = function parseReview(review, repoFullName, pullNumber) {
+   const reviewData = {
+      repo: repoFullName,
+      number: pullNumber,
+      body: review.body,
+      user: review.user,
+      created_at: review.submitted_at,
+      id: review.id,
+      state: review.state
+   }
+
+   return parseReviewSignature(reviewData);
+};
+
+function parseReviewSignature(data) {
+   let signatures = [];
+
+   if (data.state === 'APPROVED') {
```

Comment:
Given that all these ifs are mutually exclusive, I'd opt for an `if / else if` here so the code makes that obvious.